.cypress/support/commands.ts

```diff
@@ -63,6 +71,20 @@ Cypress.Commands.add('mockSearchIndexOnAction', function(
   cy.wait('@getIndices');
 });
 
+Cypress.Commands.add('mockSearchOnAction', function(
```

Comment:
looks like this is not used yet. Maybe rename to `mockSearchDetectorOnAction`?

Reply:
good catch, I was going to use it for some dashboard tests but ended up not needing it yet. I was thinking to leave it as `mockSearchOnAction` since this can be used for searching anything in ES, similar to the `searchES()` function.

Reply:
Sure. sounds good to me.